src/content/1.7/development/components/configuration.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Configuration storage
+weight: 80
+aliases:
+  - /1.7/modules/concepts/configuration/
+---
+
+# Configuration storage
+
+The Configuration storage of PrestaShop aims to address two needs.
+
+First, there are multiple situations where a single value has to be stored and be available easily. It can be a boolean to control order mode, it can be a limit for timeout: a simple key => value(s) storage is needed to manage multiple settings in PrestaShop.
+
+Secondly, modules might sometimes need a simple storage system, and they should not have to create a new SQL table for such a simple need.
+
+To address these needs, the Configuration storage of PrestaShop uses a SQL table `ps_configuration` and allows simple fetch, update and deletion of values in it.
+
+{{< figure src="../img/configuration_sql_structure.png" title="Configuration SQL structure" >}}
```

Comment:
Don't you think it would be better to store this within the components section? (If yes, you will need to move the file)

```suggestion
{{< figure src="img/configuration_sql_structure.png" title="Configuration SQL structure" >}}
```

Reply:
I'm not to understand, the PNG image is already inside `components` section.
```
src/content/1.7/development/components/img/configuration_sql_structure.png 
```

To be able to use `img/configuration_sql_structure.png` filepath we would need to create a `configuration` subdirectory inside `component` directory, and its only purpose would be to contain `img`.

is it worth it?

Reply:
Oh right, forgot it wasn't a directory.